### ISSUE TITLE
No longer needed

### ISSUE THREAD

**Issue by azack** [Jormund/split_ornaments#1]
The functionality of this script should no longer be needed as there were changes made in the base project here https://github.com/iitc-project/ingress-intel-total-conversion/pull/1144

**Comment by Jormund** [comment]
Thanks for the notice.
I'll keep this plugin for now, it hasn't been included in the release published on iitc.me 0.25.2.20161003.4740

**Comment by azack** [comment]
Yup! You can see it in the test build here if you'd like https://static.iitc.me/build/test/total-conversion-build.user.js
